auditeval/test.go

```diff
@@ -154,6 +154,7 @@ func getFlagValue(s, flag string) string {
 
 	var flagVal string
 	pttns := []string{
+		flag + ` = ([^ \n]*)`,
```

Comment:
I think this will work if there is exactly one space either side of the `=`, or no spaces, but not if there is a space before and not after, or vice versa. I think the equivalent code in kube-bench does this more neatly with a regex?

Reply:
might be better 
\s*=\s*([^ \n]*)

Reply:
looks good will check it and update